### ISSUE TITLE
it fails to parse gitlab-ci repository local images

### ISSUE THREAD

**Issue by emanuelelevo** [goodwithtech/dockle#64]
**Description**
it fails to parse gitlab-ci repository local images.

```
container_linting:
  stage: deploy
  image: docker:stable
  variables:
    DOCKER_DRIVER: overlay2
  allow_failure: true
  services:
    - docker:stable-dind
  script:
    - docker login -u gitlab-ci-token -p $CI_JOB_TOKEN $CI_REGISTRY
    - docker pull ${IMAGE_TAG}
    - VERSION=$(wget -q "https://api.github.com/repos/goodwithtech/dockle/releases/latest" -O - | grep '"tag_name":' | sed -E 's/.*"v([^"]+)".*/\1/')
    - docker run --rm goodwithtech/dockle:v${VERSION} -debug -f json ${IMAGE_TAG} > container-linting-report.json
  artifacts:
    paths: [container-linting-report.json]
```

**What did you expect to happen?**
It should find the local image.

**What happened instead?**
`2019-12-24T18:23:12.094Z	FATAL	failed to analyze image: &{%!w(string=failed to extract files: &{%!w(string=failed to fetch manifest: &{%!w(string=error unmarshalling content: invalid character '<' looking for beginning of value)})})}`

**Output of run with `-debug`:**

```
none
```

**Output of `dockle -v`:**

```
dockle version dev
```

**Additional details (base image name, container registry info...):**
```
$ docker image ls
REPOSITORY                                                  TAG                 IMAGE ID            CREATED             SIZE
git.xxxxxx.com:4567/emanuelelevo/gitlab-templates/emanuelelevo   docker-template     da293e8e44e8        52 seconds ago      121MB
$ echo $IMAGE_TAG
git.xxxxxx.com:4567/emanuelelevo/gitlab-templates/emanuelelevo:docker-template
$ docker run --rm goodwithtech/dockle:v${VERSION} ${IMAGE_TAG}
2019-12-24T18:23:12.094Z	FATAL	failed to analyze image: &{%!w(string=failed to extract files: &{%!w(string=failed to fetch manifest: &{%!w(string=error unmarshalling content: invalid character '<' looking for beginning of value)})})}
```

**Comment by tomoyamachi** [comment]
Thank you for the report! I will check it.

**Comment by DeamonMV** [comment]
Any new about it or how to check image from gitlab regestry?

**Comment by baotang2118** [comment]
This issue is fix in version 0.3.13, yep, 0.3.13 after 2 years.
In the version 0.3.11, you have to disconnect you virtual machine from the internet to scan the local images, with the internet, the tool behave unstable, sometimes can scan and sometime cannot.
In the version < 0.3.9, you cannot scan local images.
